populus/contracts/backends/filesystem.py

```diff
@@ -31,6 +35,16 @@ def get_matching_chain_definitions(web3, values):
             yield chain_definition
 
 
+def load_registrar_data(registrar_file):
```

Comment:
Basically a drop-in replacement for `json.load` which also ensures that deployments use this `ContractKeyMapping`